theano/gpuarray/tests/test_abstractconv.py

```diff
@@ -26,7 +26,7 @@ def setup_class(cls):
         # provide_shape is not used by the cuDNN impementation
         cls.provide_shape = [False]
 
-    def tcase(self, i, f, s, b, flip, provide_shape, fd=(1, 1)):
+    def tcase(self, i, f, s, b, flip, provide_shape, fd=(1, 1), u=False):
```

Comment:
It would be good to raise a SkipTest error if unshared is true. (Like for `fd`.) See also the other instances below.